spring-scaffold-cli/src/main/java/br/com/netodevel/command/template/TemplateHandler.java

```diff
@@ -3,10 +3,7 @@
 import br.com.generate.helpers.ScaffoldInfoHelper;
 import br.com.generator.core.GeneratorOptions;
 import br.com.templates_java.ComposeTemplate;
-import br.com.templates_java.config.jms_aws_sqs.EntryPointMessageGenerator;
-import br.com.templates_java.config.jms_aws_sqs.MessageListenerGenerator;
```

Comment:
There is no visible test coverage for this code, consider adding unit tests to ensure the functionality works as expected.